File: src/main/java/com/refah/walletwrapper/model/dto/BalanceDto.java
package com.refah.walletwrapper.model.dto;

import java.util.List;

public class BalanceDto {
    private List<ItemBalance> Items;
    private String Description = "transfer data";
    private String Currency = "IRR";
    private Integer TransactionType = 204;
    private Integer CashoutType = 0;
    private Integer ThirdPartyTransactionID = 0;

    public BalanceDto() {
    }

    public BalanceDto(List<ItemBalance> items) {
        Items = items;
    }

    public List<ItemBalance> getItems() {
        return Items;
    }

    public void setItems(List<ItemBalance> items) {
        Items = items;
    }

    public String getDescription() {
        return Description;
    }

    public void setDescription(String description) {
        Description = description;
    }

    public String getCurrency() {
        return Currency;
    }

    public void setCurrency(String currency) {
        Currency = currency;
    }

    public Integer getTransactionType() {
        return TransactionType;
    }

    public void setTransactionType(Integer transactionType) {
        TransactionType = transactionType;
    }

    public Integer getCashoutType() {
        return CashoutType;
    }

    public void setCashoutType(Integer cashoutType) {
        CashoutType = cashoutType;
    }

    public Integer getThirdPartyTransactionID() {
        return ThirdPartyTransactionID;
    }

    public void setThirdPartyTransactionID(Integer thirdPartyTransactionID) {
        ThirdPartyTransactionID = thirdPartyTransactionID;
    }
}

File: src/main/kotlin/com/refah/walletwrapper/model/BalanceDto.kt
package com.refah.walletwrapper.model

data class BalanceDto(
    var Items: List<ItemBalance>,
    var Description: String = "transfer data",
    val Currency: String = "IRR",
    val TransactionType: Int = 204,
    var CashoutType: Int = 0,
    var ThirdPartyTransactionID: Int = 0
)

data class ItemBalance(
    var Source: ChildItemBalance,
    var Target: ChildItemBalance,
    var Amount: Long,
    var Description: String = "transfer data"
)

data class ChildItemBalance(
    var TenantID: Long,
    var AccountKind: Int? = 1,
    var MobileNumber: String? = null,
    var AccountNumberCode: String? = null
)